lib/matplotlib/axes/_axes.py

```diff
@@ -3382,6 +3385,10 @@ def _update_dict(dictionary, rc_name, properties):
                             if ci[1] is not None:
                                 stats['cihi'] = ci[1]
 
+        # Use default zorder if none specified
+        if zorder is None:
```

Comment:
This should be moved to `bxp` and just let the `None` pass through this function.